tests/gdx-tests/src/com/badlogic/gdx/tests/gwt/GwtTestWrapper.java

```diff
@@ -668,6 +668,10 @@ public GdxTest instance () {
 			public GdxTest instance () {
 				return new PixmapPackerTest();
 			}
+		}, new Instancer() {
```

Comment:
You can also add `AlphaTest` which tests the changed method

Reply:
already there ;-)